### DISCUSSION TITLE
I can't see mysql as a Newrelic - Database section.

### DISCUSSION THREAD

**Discussion by tolgaand** [prisma/prisma#11955] [Q&A]
## Problem

Hello, I am using planetscale and prisma together in a node application.
My node application is hosted in heroku,
But I can't see my mysql server in the database section, is there a solution for this?
<img width="366" alt="Newrelic database section" src="https://user-images.githubusercontent.com/26303690/154927143-6492b797-6f87-400d-b70a-3e803f8c6f36.png">

## Suggested solution

I don't know which library should solve this problem.

**Comment by nurul3101** [comment]
Hey @tolgaand 👋🏻 ,
Can you please elaborate on your issue?
Are you not able to see your database in PlanetScale app?

**Reply by vladi-stevanovic** [reply]
👋 Hello @tolgaand  I'm going through pending GitHub Discussions to check if you need any help. Is this issue still open?
Please let us know if we can investigate it further or if you've already solved it, we'd love to know how!

It would be super useful for other users facing this problem! 🙌

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
